### PR TITLE
Samples for controlling clocked SPI RGB LED strips like the LPD8806 or APA102 (DotStar) from an IoT Core device.

### PR DESCRIPTION
**Pull request by superlime**

### CODE REVIEW

File: RgbLedStrip/CS/lpd8806/Lpd8806Sample/Package.appxmanifest [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  <Identity
+    Name="4a5f3a64-4386-4181-96a8-5d487c365bba"
+    Publisher="CN=scbrad"
+    Version="1.0.0.0" />
+
+  <mp:PhoneIdentity PhoneProductId="4a5f3a64-4386-4181-96a8-5d487c365bba" PhonePublisherId="00000000-0000-0000-0000-000000000000"/>
+
+  <Properties>
+    <DisplayName>Lpd8806Strip</DisplayName>
+    <PublisherDisplayName>scbrad</PublisherDisplayName>
```
Comment (ooeygui) [inline RgbLedStrip/CS/lpd8806/Lpd8806Sample/Package.appxmanifest]: MS?

File: RgbLedStrip/CS/lpd8806/Lpd8806Sample/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -0,0 +1,28 @@
+﻿using System.Reflection;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("Lpd8806Sample")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
```
Comment (ooeygui) [inline RgbLedStrip/CS/lpd8806/Lpd8806Sample/Properties/AssemblyInfo.cs:10]: MS?

File: RgbLedStrip/CS/lpd8806/Lpd8806Sample/Package.appxmanifest [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+<Package
+  xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10"
+  xmlns:mp="http://schemas.microsoft.com/appx/2014/phone/manifest"
+  xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10"
+  IgnorableNamespaces="uap mp">
+
+  <Identity
+    Name="4a5f3a64-4386-4181-96a8-5d487c365bba"
+    Publisher="CN=scbrad"
```
Comment (bfjelds) [inline RgbLedStrip/CS/lpd8806/Lpd8806Sample/Package.appxmanifest]: publisher should be some form of msft

File: RgbLedStrip/CS/apa102/DotStarSample/Package.appxmanifest [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+<Package
+  xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10"
+  xmlns:mp="http://schemas.microsoft.com/appx/2014/phone/manifest"
+  xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10"
+  IgnorableNamespaces="uap mp">
+
+  <Identity
+    Name="4a5f3a64-4386-4181-96a8-5d487c365bba"
+    Publisher="CN=scbrad"
```
Comment (bfjelds) [inline RgbLedStrip/CS/apa102/DotStarSample/Package.appxmanifest]: publisher should be some form of msft

File: RgbLedStrip/CS/apa102/DotStarSample/Package.appxmanifest [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+    Version="1.0.0.0" />
+
+  <mp:PhoneIdentity PhoneProductId="4a5f3a64-4386-4181-96a8-5d487c365bba" PhonePublisherId="00000000-0000-0000-0000-000000000000"/>
+
+  <Properties>
+    <DisplayName>DotStarStrip</DisplayName>
+    <PublisherDisplayName>scbrad</PublisherDisplayName>
```
Comment (bfjelds) [inline RgbLedStrip/CS/apa102/DotStarSample/Package.appxmanifest]: publisher should be some form of msft

**Comment by ooeygui** [comment]
Hold off on pulling this.

**Comment by superlime** [comment]
Sorry for the delay!  Was out on vacation.  Publisher info should be fixed now.  :)

**Comment by bfjelds** [comment]
please add license.txt to folder and copyright banner to source code (see other samples)

**Pull request merged**